CHANGELOG.md

```diff
@@ -22,6 +28,10 @@ The present file will list all changes made to the project; according to the
 - Remove `$CFG_GLPI['use_rich_text']` parameter. Will now be `true` per default.
 - Remove `$CFG_GLPI['ticket_timeline']` parameter. Will now be `true` per default.
 - Remove `$CFG_GLPI['ticket_timeline_keep_replaced_tabs']` parameter. Will now be `false` per default.
+- Move `TicketFollowup::showFormMassiveAction` to `ITILFollowup` and deprecate.
```

Comment:
`show*` and massive no longer exists at all (and there is no replacement); no depreciation message will never be shown... Well do not see how to manage this nightmare

We've discussed about this "dead code"; it will be handled more globally in another PR;

Reply:
I just put back empty TicketFollowup class to handle deprecation state.